server.go

```diff
@@ -27,6 +27,7 @@ type Server struct {
 	PasswordHandler             PasswordHandler             // password authentication handler
 	PublicKeyHandler            PublicKeyHandler            // public key authentication handler
 	PtyCallback                 PtyCallback                 // callback for allowing PTY sessions, allows all if nil
+  ConnCallback                ConnCallback                // optional callback for wrapping net.Conn before handling
```

Comment:
Just noticed this... looks like it's indented with spaces? A gofmt might be prudent.